packages/adminrouter/extra/src/README.md

```diff
@@ -431,6 +432,9 @@ frameworks that:
 * become disconnected from Mesos for some reason, but *MAY* come back at some
   point
 
+Note that in DC/OS Enterprise  only frameworks that are started by Root 
```

Comment:
There is a double space after 'Enterprise'. I think that this is meant to be ", ".

Reply:
fixed